File: ase/misc/cpnt.js

var c = Native.ComponentTest;
var x = c();
for (var i = 0; i < 3; ++i) {
  var r = x.foo(10);
  var r = x.bar(10);
  var r = x.baz(10);
  var r = c.hoge(10);
  var r = c.fuga(10);
}
print(r);

File: ase/misc/cpnt.lua

local c = Native.ComponentTest
local x = c()
local i, r
for i = 0, 2 do
  r = x.foo(10)
  r = x.bar(10)
  r = x.baz(10)
  r = c.hoge(10)
  r = c.fuga(10)
end
print(r)

